### ISSUE TITLE
Mention `make venv` on Helping with Documentation page

### ISSUE THREAD

**Issue by csabella** [python/devguide#334]
I asked a question on core-mentorship because I wasn't able to build the docs with `make html` as per https://devguide.python.org/docquality/#helping-with-the-developer-s-guide.  Ned replied to do `make venv`, then `make html`, which worked.  Does the devguide need to be updated for this or was it just my environment was stale?

Thanks!

**Comment by ned-deily** [comment]
An update would be good.  This is another case of DRY; perhaps you could take a look at not how to not repeat this.  It *is* documented in the devguide here: https://devguide.python.org/documenting/#using-make-make-bat.  And we have had some discussions about removing the "Without make" section there.

**Comment by Mariatta** [comment]
Removing the without make section is being done in #327.

**Comment by csabella** [comment]
I made a mistake in my original post.  I started looking at updating the devguide per Ned's suggestions and realized my original link was not to how to build the python docs, but rather how to build the devguide.  I think this is very confusing because those instructions are contained on the "Helping with Documentation" page and that page is intended for new contributors (it's only linked to from the "Beginner's" section on the index page and the index itself).  

Just thinking of workflow for a new contributor - 
- Go to Github and look at README
- Second bullet is 'Contributing', so click there.
- That leads to the Dev Guide index.
- Which leads to the contributing section of the index page.
- Under beginner tasks, leads to "Helping with Documentation"

Based on that workflow, I've made a PR that I think will help with the "Helping with Documentation" page and it's link to "Documenting Python".  I don't know if "Helping with the Developer's Guide" should even be mentioned on that page, but I don't know where else it would live.
